tools/nnyeah/nnyeah/AssemblyComparator/ComparingVisitor.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.MaciOS.Nnyeah;
+using Mono.Cecil;
+
+#nullable enable
+
+namespace Microsoft.MaciOS.AssemblyComparator {
```

Comment:
In the PR I also fixed a number of problems with the namespaces, which are wrong, we should fix those too else most IDEs will throw warnings and there are a few not needed uings.